wiki/Help_centre/Upgrading_to_lazer/it.md

```diff
@@ -0,0 +1,290 @@
+
+#### Le vecchie skin funzioneranno nella selezione delle mappe e nella schermata del risultato?
+
+Faremo del nostro meglio per riportare queste funzionalità
+
+#### Posso usare il cursore della skin pure sul menù?
+
+Porteremo molto probabilmente questa novità date la richiesta elevata.
+
+### Prestazioni
+
+#### Perchè non posso avere gli fps illimitati?
+
+Al di sopra di una certa soglia, non c'è motivo di utilizzare frame rate più elevati. Lazer impiega diverse nuove tecnologie per garantire la latenza più bassa possibile senza richiedere frame rate elevati. Questo aspetto continuerà a migliorare in futuro, poiché abbiamo altre migliorie da implementare.
+
+Lazer prende l'input a 1000Hz indipendentemente dal limitatore di FPS, che è anche il motivo per cui l'impostazione massima del limitatore è 1000 FPS.
+
+Se siete curiosi di sapere come questo influisce sulla latenza in ingresso e di verificare la vostra percezione, eseguite il "Run latency certifier" integrato in fondo alle impostazioni.
+
+Potresti anche [leggere questo documento tecnico](https://github.com/ppy/osu/wiki/Latency-and-unlimited-frame-rates) in cui viene spiegato il percorso che stiamo intraprendendo e le motivazioni dietro alle decisioni prese.
+
+#### Se il polling dell'input viene eseguito solo a 1000 Hz, cosa succede al mio mouse da gioco a 8000 Hz?
+
+Il sistema operativo continuerà a eseguire il polling alla frequenza più alta possibile, anche se i vantaggi sono trascurabili. Il polling a potenza così elevata può comportare richiesta di potenza aggiuntiva e si consiglia di limitare le proprie periferiche a 1000 Hz per garantire la stabilità.
+
+#### Lazer performa peggio di stable per me. Che succede?
+
+Sebbene sulla maggior parte dell'hardware moderno notiamo che lazer supera stable, però, ci sono sempre casi dove utenti non hanno una configurazione hardware performante. Nella nostra roadmap a breve termine, stiamo cercando di supportare DirectX (alias "modalità compatibile" su stable) e Vulkan, che hanno entrambi un supporto driver migliore di OpenGL su tutto l'hardware. Una volta implementato, le prestazioni su hardware come i chipset integrati Intel miglioreranno notevolmente.
+
+### Fornire feedback
+
+#### Manca una funzione da cui dipendo! / Qualcosa è cambiato e non mi piace. / Ho trovato un bug, qual è il modo migliore per segnalarlo?
+
+È molto probabile che ne siamo già a conoscenza e che lo stiamo monitorando per una futura implementazione! Cerca nell'[issue tracker](https://github.com/ppy/osu/issues) e nella [pagina delle discussioni](https://github.com/ppy/osu/discussions). Se non trovate nessuna discussione che corrisponde al vostro problema, sentitevi liberi di [aprirne una nuova](https://github.com/ppy/osu/discussions/new).
+
+Si noti che stiamo già monitorando più di 1.000 problemi più o meno critici e che potrebbe volerci del tempo per risolvere problemi che riguardano solo un piccolo numero di utenti.
+
+### Altro
```

Comment:
this is missing the "Why is it called "lazer"?" question. see the [french version](https://github.com/ppy/osu-wiki/pull/8481/files#diff-9cffabcd9c5de51fe9f9981448dc8188941dbb4dd8cff7fb724bc53d9a42647fR217-R219) for an example on how it could be translated